script/app-question-classification/_cm.json

```diff
@@ -2,9 +2,19 @@
   "alias": "app-question-classification",
   "automation_alias": "script",
   "automation_uid": "5b4e0237da074764",
-  "cache": false,
+  "cache": true,
```

Comment:
Why we need cache here?

Reply:
I changed it to true when I had to work on error in preprocessing script so that the inference script won't run again costing amount from API.